assets/queries/openAPI/2.0/security_definitions_using_password_flow/query.rego

```diff
@@ -6,14 +6,14 @@ CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) == "2.0"
 
-	secDef := doc.securityDefinitions[key]
-	secDef.flow == "password"
+	doc.securityDefinitions[key].flow == "password"
+	doc.securityDefinitions[key].type == "oauth2"
```

Comment:
Based on [this comment](https://github.com/Checkmarx/kics/pull/3664#discussion_r652498677), I think `doc.securityDefinitions[key].type == "oauth2"` is not necessary